File: src/main/java/core/data/report/DataTableAlgorithmReport.java
package core.data.report;

import core.data.DataTable;

/**
 * MIRF report of {@link core.algorithm.Algorithm} presented as data table
 */
public class DataTableAlgorithmReport extends AlgorithmReport {

    public DataTable table;

    public DataTableAlgorithmReport(DataTable table) {
        this.table = table;
        mirfReportType = MirfReportType.DataTable;
    }
}

File: src/main/java/core/data/report/DataTableAlgorithmReport.kt
package core.data.report

import core.data.DataTable

/**
 * MIRF report of [core.algorithm.Algorithm] presented as data table
 */
class DataTableAlgorithmReport(var table: DataTable) : AlgorithmReport() {

    init {
        mirfReportType = MirfReportType.DataTable
    }
}
